projects/kubernetes/kubernetes/Dockerfile

```diff
@@ -28,15 +56,19 @@ RUN apk add --no-cache --initdb -p /out \
     socat \
     util-linux \
     && true
+
+RUN cp _output/bin/kubelet /out/usr/bin/kubelet
+RUN cp _output/bin/kubeadm /out/usr/bin/kubeadm
+RUN cp _output/bin/kubectl /out/usr/bin/kubectl
+
+RUN strip /out/usr/bin/kubelet /out/usr/bin/kubeadm /out/usr/bin/kubectl
```

Comment:
You also strip above, once should be enough

Reply:
Dang, I _knew_ I did it on Friday but missed it when I checked so I added it again this morning, Doh!

Reply:
I don't believe it is safe to use `strip` on Go binaries. You can use the Go build options that we use in our build script that does a partial `strip`.

Reply:
What are the symptoms? These ones worked well enough to run the sock shop demo, but perhaps the issues are subtle?